.env

```diff
@@ -66,3 +69,7 @@ PRICES_DB_PASSWORD=magic
 
 PRICES_SERVICE_URI=prices.mainnet.teritori.com:443
 
+#PINATA
+PINATA_API_KEY='295bb1b4e03e102493e4'
+PINATA_SECRET_API_KEY='acdecdb3a453b27ec76092dec2a455eba19da897cbff828dbac027f83b79fd74'
```

Comment:
@0xoscario this one should be private?

Reply:
Yes, need to use the backend way, as with Social Feed